src/intl/messages.js

```diff
@@ -262,6 +262,7 @@ export const messages = {
   network: 'Network',
   noNetwork: 'No network',
   vnicProfileEmpty: '<Empty>',
+  bootMenuWarning: 'All changes will take effect only after reboot!',
```

Comment:
Please no exclamation mark `!` in similar texts.
Considering time constraint, let's keep it for a follow-up